frontend/src/components/BottomNavigationToDelete.js

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import CssBaseline from '@mui/material/CssBaseline';
+import BottomNavigation from '@mui/material/BottomNavigation';
+import BottomNavigationAction from '@mui/material/BottomNavigationAction';
+import RestoreIcon from '@mui/icons-material/Restore';
+import FavoriteIcon from '@mui/icons-material/Favorite';
+import ArchiveIcon from '@mui/icons-material/Archive';
+import Paper from '@mui/material/Paper';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemAvatar from '@mui/material/ListItemAvatar';
+import ListItemText from '@mui/material/ListItemText';
+import Avatar from '@mui/material/Avatar';
+
+function refreshMessages() {
+    const getRandomInt = (max) => Math.floor(Math.random() * Math.floor(max));
+
+    return Array.from(new Array(50)).map(
+        () => messageExamples[getRandomInt(messageExamples.length)],
+    );
+}
+
+export default function FixedBottomNavigation() {
+    const [value, setValue] = React.useState(0);
+    const ref = React.useRef(null);
+    const [messages, setMessages] = React.useState(() => refreshMessages());
```

Comment:
You dont need the React Keyword
```suggestion
    const [value, setValue] = useState(0);
    const ref = useRef(null);
    const [messages, setMessages] = useState(() => refreshMessages());
```